library/src/main/java/org/polaric/colorful/Colorful.java

```diff
@@ -7,23 +7,28 @@
 
 public class Colorful {
     private static ThemeDelegate delegate;
-    private static ThemeColor primaryColor= ThemeColor.DEEP_PURPLE;
-    private static ThemeColor accentColor= ThemeColor.RED;
-    private static boolean isTranslucent=false;
-    private static boolean isDark=false;
+    private static ThemeColor primaryColor = Defaults.primaryColor;
```

Comment:
I think its discussable if the default values should be in `Defaults` or `Colorful` but it's not DRY to have them in both classes